rendermodules/pointmatch/generate_point_matches_spark.py

```diff
@@ -116,7 +117,9 @@ def get_pointmatch_args(cls, baseDataUrl=None, owner=None,
             get_cmd_opt(renderWithoutMask, '--renderWithoutMask') +
             get_cmd_opt(renderFullScaleWidth, '--renderFullScaleWidth') +
             get_cmd_opt(renderFullScaleHeight, '--renderFullScaleHeight') +
-            get_cmd_opt(fillWithNoise, '--fillWithNoise'))
+            get_cmd_opt(fillWithNoise, '--fillWithNoise')+
+            get_cmd_opt(rootFeatureDirectory,'--rootFeatureDirectory')+
+            get_cmd_opt(requireStoredFeature,'--requireStoredFeature'))
```

Comment:
requireStoredFeature is an arity 0 argument which is supported by `cls.get_flag_cmd`.